components/camera/replaypcd/replaypcd.go

```diff
@@ -89,6 +91,12 @@ func (cfg *Config) Validate(path string) ([]string, error) {
 	if cfg.OrganizationID == "" {
 		return nil, goutils.NewConfigValidationFieldRequiredError(path, "organization_id")
 	}
+	if cfg.APIKey == "" {
+		return nil, goutils.NewConfigValidationFieldRequiredError(path, "api_key")
+	}
+	if cfg.APIKeyID == "" {
+		return nil, goutils.NewConfigValidationFieldRequiredError(path, "api_key_id")
+	}
```

Comment:
thought about having this pr not require api keys to make it easier to configure a replay sensor locally, but because of the upcoming robot secret permission changes I figured it was better to just require api keys